APE14.rst

```diff
@@ -467,6 +638,33 @@ for ``pixel_to_world``, but on the other hand provided there is no ambiguity,
 the wrong order (though an error should be raised if there are any ambiguities
 and the order is not the standard one).
 
+Note that the low- and high-level APIs will be defined as base classes that
+will be designed in such a way that a custom WCS class can inherit from both
+the low- and high-level base classes. The high-level base class will be
+implemented with methods that will by default use the information in
+``world_axis_object_components`` and ``world_axis_object_classes`` to work -
+thus, simply inheriting from the high-level base class should be sufficient to
+expose the high-level API.
+
+High-level Astropy Object
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Finally, we will develop a class in Astropy that inherits from the high-level
```

Comment:
"Astropy" -> \`astropy\` (i.e., the core package).

Reply:
Fixed